src/client/components/styled-components/index.js

```diff
@@ -103,6 +103,20 @@ export const DropDownWithValidation = props => (
 DropDownWithValidation.propTypes = { error: PropTypes.string }
 DropDownWithValidation.defaultProps = { error: '' }
 
+export const SchoolCompanyName = props => (
```

Comment:
Maybe `CardHeader` would be a better name?

Reply:
Much better! Although it's TileHeader, Card contains multiple Tiles, and this is single Tile header.

Reply:
Second point: yes, we usually do "squash and merge" in github while merging.